### DISCUSSION TITLE
How to sponser Lailloken for this amazing POE app???

### DISCUSSION THREAD

**Discussion by FaLLeNaNg3L82** [Lailloken/Exile-UI#402] [Q&A]
I would like to contribute financially for the amazing work you did with the UI helped me a TON!!! 
Couldnt find a sponsor button.

**Comment by Lailloken** [comment]
Thank you very much for the kind gesture, I'll will think about adding ways to contribute.
I have not been taking contributions because this project was mostly about having fun and exploring the world of coding, but more and more people have recently expressed their wish to contribute, so I'll explore some options.

Again, I appreciate the gesture.
